sdk/appcenter-distribute/src/test/java/com/microsoft/appcenter/distribute/DistributeDisableAutomaticCheckForUpdateTest.java

```diff
@@ -61,4 +62,65 @@ public void manualCheckWhileAutomaticCheckDisabledChecksForUpdate() {
         /* HTTP call done. */
         verify(mHttpClient).callAsync(anyString(), anyString(), eq(Collections.<String, String>emptyMap()), any(HttpClient.CallTemplate.class), any(ServiceCallback.class));
     }
+
+    @Test
+    public void manualCheckAfterActivityResumedWhileAutomaticCheckDisabledChecksForUpdate() {
+
+        /* Start then call disable automatic check for update after Distribute has started. */
+        Distribute.disableAutomaticCheckForUpdate();
+        start();
+        verify(mHttpClient, never()).callAsync(anyString(), anyString(), eq(Collections.<String, String>emptyMap()), any(HttpClient.CallTemplate.class), any(ServiceCallback.class));
+        Distribute.getInstance().onActivityResumed(mActivity);
+        Distribute.checkForUpdate();
+
+        /* HTTP call done. */
+        verify(mHttpClient).callAsync(anyString(), anyString(), eq(Collections.<String, String>emptyMap()), any(HttpClient.CallTemplate.class), any(ServiceCallback.class));
+    }
+
+    @Test
+    public void checkForUpdateAfterManualCheckCompletesDoesNotCheckForUpdatesAgain() {
+
+        /* Start in public mode with automatic check for updates disabled. */
+        Distribute.disableAutomaticCheckForUpdate();
+        start();
+        Distribute.getInstance().onActivityResumed(mActivity);
+        Distribute.checkForUpdate();
```

Comment:
We also need a variant of this test where we swap lines 86 and 87 and we verify the first call is also made (no need to repeat the second part of the test).

Reply:
Isn't that just the same as the test on line 53?  Is there a difference between that test and the one you're describing?

Reply:
oh I didn't realize that.